crates/astria-conductor/src/conductor.rs

```diff
@@ -83,22 +82,26 @@ impl Conductor {
         let signals = spawn_signal_handler();
 
         // Spawn the executor task.
-        let executor_tx = {
+        let (executor_tx, init_sequencer_height) = {
             let (block_tx, block_rx) = mpsc::unbounded_channel();
             let (shutdown_tx, shutdown_rx) = oneshot::channel();
             let executor = Executor::new(
                 &cfg.execution_rpc_url,
                 ChainId::new(cfg.chain_id.as_bytes().to_vec())
                     .wrap_err("failed to create chain ID")?,
                 cfg.disable_empty_block_execution,
+                cfg.initial_sequencer_block_height,
                 block_rx,
                 shutdown_rx,
             )
             .await
             .wrap_err("failed to construct executor")?;
+            let init_sequencer_height = executor.executable_block_height;
```

Comment:
name this `executable_sequencer_block_height` or something as it isn't the `initial_sequencer_block_height`